operator/gitops/compute/pipeline-service-manager/role.yaml

```diff
@@ -5,12 +5,106 @@ metadata:
   name: pipeline-service-admin
 rules:
   - apiGroups:
+      - ""
+    resources:
+      - namespaces
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - patch
+      - delete
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+      - secrets
+      - serviceaccounts
+      - services
+    verbs:
       - "*"
+  - apiGroups:
+      - apps
     resources:
       - "*"
     verbs:
       - "*"
-  - nonResourceURLs:
+  - apiGroups:
+      - apiextensions.k8s.io
+    resources:
+      - customresourcedefinitions
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - networking.k8s.io
+    resources:
+      - ingress
+      - networkpolicies
+    verbs:
```

Comment:
This uses wildcard permissions on ingress and network policies. Are we sure we need to grant this level of elevated permission?

Reply:
Yes, it seems to be one of the required privileges for the kcp-syncer to function properly.